lib/ui/window.dart

```diff
@@ -225,6 +225,32 @@ abstract class FlutterView {
   ///   applications.
   WindowPadding get padding => viewConfiguration.padding;
 
+  /// Areas of the display that are obstructed by hardware features.
```

Comment:
Since this seems to be repeated docs, you may want to use the templates system to avoid repeating documentation that is the same in multiple places